src/domain/usecases/ClosePatientsUseCase.ts

```diff
@@ -25,16 +25,43 @@ export class ClosePatientsUseCase {
 
         const payload = await this.programsRepository
             .get({ programId, orgUnitsIds, startDate, endDate })
-            .then(trackedEntities =>
-                this.filterEntities(
-                    trackedEntities,
+            .then(async trackedEntities => ({
+                teis: await this.programsRepository.getTeis(trackedEntities.map(tei => tei.trackedEntity)), //TEMPORAL FIX
```

Comment:
As it's a DHIS2 bug, the workaround should be done internally in ProgramsD2Repository, the use case should not care about it.